SwanProjects/bin/swan_kmspecs

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python
+import argparse
+import json
+import os
+import pprint
+import subprocess
+import sys
+from distutils.spawn import find_executable
+from shutil import rmtree
+
+def _checkipykernel(project_path,python_interpreter,python_version):
```

Comment:
Why is `_checkipykernel` flagged as internal with `_` but not the others?

Reply:
fixed